src/ffi/spatialmath/vector3.rs

```diff
@@ -13,42 +13,42 @@ pub unsafe extern "C" fn free_vector_memory(ptr: *mut Vector3) {
     if ptr.is_null() {
         return;
     }
-    Box::from_raw(ptr);
+    let _ = Box::from_raw(ptr);
```

Comment:
Same question here: why did this line change?

Reply:
see above re: clippy.